units/systemd-udevd.service.in

```diff
@@ -13,6 +13,7 @@ Documentation=man:systemd-udevd.service(8) man:udev(7)
 DefaultDependencies=no
 After=systemd-sysusers.service systemd-hwdb-update.service
 Before=sysinit.target
+Wants=systemd-udev-load-credentials.service
```

Comment:
Hmm, it's not necessary to make `systemd-udev-load-credentials.service` installable then?

Reply:
While I create this PR, I surprised that systemd-udevd.service does not have [Install] section. I do not know why it does not have that... If udevd has [Install] section, then I think it is better to set `Also=` there...
@poettering Could you give me a hint??

Reply:
udevd is statically enabled, no? Also, `dbus.service` doesn't have `[Install]` section either.

So this is not something we expect user to manually enable, so I don't think `Also=` is the right choice.

Reply:
I see. Yeah, but the service should be run anyway before systemd-udevd.service unconditionally. If we drop the line, even if we set [Install] section for the new service, the service must be enabled manually or by preset. preset may not be used by all distribution or on upgrade, though I am not sure. So, setting `Wants=` in the main udevd.service should be the best approach, I think.

Reply:
I mean, I like the line, so I think `[Install]` in the other unit is not necessary :)